doc/manual/book.toml

```
@@ -1,6 +1,11 @@
+[book]
+title = "Nix Package Manager Manual"
```

```suggestion
title = "Nix Reference Manual"
```

As [discussed on the team meeting](https://github.com/NixOS/nix/pull/7300#issuecomment-1332911864).